File: Modul-4/soal-3/PRAK403_2410817210017_OrlandoSugian.py
datainput = (input(""))
digit1, digit2 = map(int,datainput.split())
if digit1 < digit2 : 
    for i in range (digit1, digit2 + 1) :
        print(f"{i} {digit1 + digit2 -i}", end=" - " if i < digit2 else "")
else : 
    for i in range (digit1, digit2 - 1, -1):
        print(f"{i} {digit1 + digit2 -i}", end=" - " if i > digit2 else "")

File: Modul-4/soal-3/PRAK403_2410817210017_OrlandoSugian.c
#include<stdio.h>
int main()
{
    int digit1, digit2;
    scanf("%d %d", &digit1, &digit2);
    if (digit1 < digit2){
        for (int i = digit1; i <= digit2; i++){
            printf("%d %d", i, digit1 + digit2 - i);
            if (i != digit2) 
            printf(" - ");
        }
    } else {
        for ( int i = digit1; i >= digit2; i--) {
            printf("%d %d", i, digit1 + digit2 -i);
        if (i != digit2) 
            printf(" - ");
        }
    } return 0;
}